docs/conf.py

```diff
@@ -120,12 +126,12 @@
             "icon": "fab fa-github-square",
         },
     ],
+    "logo": {
+        "image_light": "PreliZ.png",
+        "image_dark": "PreliZ_white.png",
```

Comment:
```suggestion
        "image_light": "PreliZ_flat.png",
        "image_dark": "PreliZ_flat_white.png",
```

I think this will look much better given the theme

Reply:
Sorry, I saw this after doing the last commit